R/generateSequenceCohortSet.R

```diff
@@ -203,10 +203,11 @@ generateSequenceCohortSet <- function(cdm,
 
   # 3) days prior observation
   cdm[[name]] <- cdm[[name]] %>%
-    dplyr::filter(
-      .data$prior_observation_marker >= .env$daysPriorObservation &
-      .data$prior_observation_index >= .env$daysPriorObservation
-    )
+    dplyr::mutate(min_obs = dplyr::if_else(.data$prior_observation_index <= .data$prior_observation_marker,
```

Comment:
why not just do the filter relative to cohort_start_date?

Reply:
Ah good point @edward-burn!  I suppose we can't do this trick for washout right? https://github.com/oxford-pharmacoepi/CohortSymmetry/blob/main/R/generateSequenceCohortSet.R#L213C3-L217C6 
I am not so sure on how to make this bit simpler? Thanks!